templates/core/core-pre-upgrade-job.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.core.enableMigrateHelmHook }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "harbor.core" . }}-job
+  labels:
+{{ include "harbor.labels" . | indent 4 }}
+    component: core
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-weight": "-5"
+spec:
+  template:
+    metadata:
+      labels:
+{{ include "harbor.matchLabels" . | indent 8 }}
+        component: core-job
+      annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/core/core-cm.yaml") . | sha256sum }}
```

Comment:
this checksum was introduced to control the recreation of the pod, so probably does not make sense for the job?

Reply:
removed